actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+module ActionText
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+
+      def install_javascript_dependencies
+        template "#{GEM_ROOT}/../railties/lib/rails/generators/rails/app/templates/bin/yarn.tt",
```

Comment:
Again, I doubt this will work once Action Text is packaged as a gem. Try running `gem package` locally (or build or whatever the command is called). Then I think you can bundle the gem by pointing to it with `:path`. Finally try running `actiontext:install` from within your test app and see if it works.

Reply:
How about changing the template copying to

```ruby
run "rake app:update:bin"
```

This will just create or update all bin files. Which would make sure `bin/yarn` exists. What do you think?

Reply:
I'm a little hesitant but don't know why. Let's go with it for now. @y-yagi if you can see anything wrong with this do follow up later 🙏